src/Nox.Types/Types/Color/KnownColor.cs

```diff
@@ -0,0 +1,164 @@
+        { "MediumVioletRed", System.Drawing.Color.MediumVioletRed },
+        { "MidnightBlue", System.Drawing.Color.MidnightBlue },
+        { "MintCream", System.Drawing.Color.MintCream },
+        { "MistyRose", System.Drawing.Color.MistyRose },
+        { "Moccasin", System.Drawing.Color.Moccasin },
+        { "NavajoWhite", System.Drawing.Color.NavajoWhite },
+        { "Navy", System.Drawing.Color.Navy },
+        { "OldLace", System.Drawing.Color.OldLace },
+        { "Olive", System.Drawing.Color.Olive },
+        { "OliveDrab", System.Drawing.Color.OliveDrab },
+        { "Orange", System.Drawing.Color.Orange },
+        { "OrangeRed", System.Drawing.Color.OrangeRed },
+        { "Orchid", System.Drawing.Color.Orchid },
+        { "PaleGoldenrod", System.Drawing.Color.PaleGoldenrod },
+        { "PaleGreen", System.Drawing.Color.PaleGreen },
+        { "PaleTurquoise", System.Drawing.Color.PaleTurquoise },
+        { "PaleVioletRed", System.Drawing.Color.PaleVioletRed },
+        { "PapayaWhip", System.Drawing.Color.PapayaWhip },
+        { "PeachPuff", System.Drawing.Color.PeachPuff },
+        { "Peru", System.Drawing.Color.Peru },
+        { "Pink", System.Drawing.Color.Pink },
+        { "Plum", System.Drawing.Color.Plum },
+        { "PowderBlue", System.Drawing.Color.PowderBlue },
+        { "Purple", System.Drawing.Color.Purple },
+        { "Red", System.Drawing.Color.Red },
+        { "RosyBrown", System.Drawing.Color.RosyBrown },
+        { "RoyalBlue", System.Drawing.Color.RoyalBlue },
+        { "SaddleBrown", System.Drawing.Color.SaddleBrown },
+        { "Salmon", System.Drawing.Color.Salmon },
+        { "SandyBrown", System.Drawing.Color.SandyBrown },
+        { "SeaGreen", System.Drawing.Color.SeaGreen },
+        { "SeaShell", System.Drawing.Color.SeaShell },
+        { "Sienna", System.Drawing.Color.Sienna },
+        { "Silver", System.Drawing.Color.Silver },
+        { "SkyBlue", System.Drawing.Color.SkyBlue },
+        { "SlateBlue", System.Drawing.Color.SlateBlue },
+        { "SlateGray", System.Drawing.Color.SlateGray },
+        { "Snow", System.Drawing.Color.Snow },
+        { "SpringGreen", System.Drawing.Color.SpringGreen },
+        { "SteelBlue", System.Drawing.Color.SteelBlue },
+        { "Tan", System.Drawing.Color.Tan },
+        { "Teal", System.Drawing.Color.Teal },
+        { "Thistle", System.Drawing.Color.Thistle },
+        { "Tomato", System.Drawing.Color.Tomato },
+        { "Turquoise", System.Drawing.Color.Turquoise },
+        { "Violet", System.Drawing.Color.Violet },
+        { "Wheat", System.Drawing.Color.Wheat },
+        { "White", System.Drawing.Color.White },
+        { "WhiteSmoke", System.Drawing.Color.WhiteSmoke },
+        { "Yellow", System.Drawing.Color.Yellow },
+        { "YellowGreen", System.Drawing.Color.YellowGreen },
+    };
+
+    public static bool TryGetNamedColor(string text,  out System.Drawing.Color color)
```

Comment:
this can be replace by this: `Enum.TryParse<System.Drawing.Color>(text, out color);`

Reply:
Promote this methodo to ColorHelper